os/net/bluetooth/bluetooth.c

```diff
@@ -1042,20 +1042,28 @@ void bt_foreach_bond(uint8_t id, void (*func)(const struct bt_bond_info *info, v
 
 struct bt_conn *bt_conn_ref(struct bt_conn *conn)
 {
-	// TODO: need to implement
-	return NULL;
+	struct bt_conn_s *conns;
+
+	conns = bt_conn_addref((struct bt_conn_s*)conn);
+
+	return (struct bt_conn*)conns;
 }
 
 void bt_conn_unref(struct bt_conn *conn)
 {
-	// TODO: need to implement
+	bt_conn_relref((struct bt_conn_s *)conn);
+
 	return;
 }
 
 struct bt_conn *bt_conn_lookup_addr_le(uint8_t id, const bt_addr_le_t *peer)
 {
-	// TODO: need to implement
-	return NULL;
+	struct bt_conn_s * conns;
+
+	conns = bt_conn_lookup_addr_le_id(id, peer);
```

Comment:
I think we don't need to add this new function.
we can do that in here.

Reply:
We are using bt_conn_lookup_addr_le_id() at some other places also.